org.kde.kcolorchooser.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "org.kde.kcolorchooser",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kcolorchooser",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "kcolorchooser",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/kcolorchooser-21.12.1.tar.xz",
+                    "sha256": "a3b34c3fe699435664f99a5fad7960f7b38147ecd17010faafd3d03e49cabeb2"
+                },
+                {
+                    "type": "patch",
+                    "path": "appdata.patch"
+                }
+            ],
+            "post-install": [
+                "mkdir -p /app/share/icons/hicolor/scalable/apps",
+                "install -D /usr/share/icons/breeze/apps/48/kcolorchooser.svg /app/share/icons/hicolor/scalable/apps/"
```

Comment:
```suggestion
                "install -D /usr/share/icons/breeze/apps/48/kcolorchooser.svg /app/share/icons/hicolor/scalable/apps/kcolorchooser.svg"
```

Reply:
Understood :+1: 

I've made changes and pushed. Thanks!